mtp_noms_ops/apps/security/urls.py

```diff
@@ -3,8 +3,23 @@
 from . import required_permissions, views
 from mtp_noms_ops.view_utils import user_test
 
-
+security_test = user_test(required_permissions)
 urlpatterns = [
-    url(r'^$', user_test(required_permissions)(views.SecurityDashboardView.as_view()),
+    url(r'^$', security_test(views.dashboard_view),
         name='security_dashboard'),
+
+    url(r'^sender-grouped/$', security_test(views.SenderGroupedView.as_view()),
+        name='security_sender_grouped'),
```

Comment:
Would it be easier to use Django's in-built URL namespacing rather than manually prefixing all these URL names with ```security_```?